### PR TITLE
Add FR locale.xml

### PR DESCRIPTION
**Pull request by bastoune**
* [x] add locale/fr_FR/locale.xml

### CODE REVIEW

File: locale/fr_FR/locale.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	<message key="editor.issues.sectionArea">Options pour les sections des numéros</message>
+
+	<!-- General front-end -->
+	<message key="plugins.themes.immersion.issue.published">Publié</message>
+	<message key="plugins.themes.immersion.issue.description">Description du numéro</message>
+	<message key="plugins.themes.immersion.registration.consent">Consentement</message>
+	<message key="plugins.themes.immersion.mainMenu">Menu principal</message>
+	<message key="plugins.themes.immersion.language.toggle">Changer de langue. La langue actuelle est:</message>
+	<message key="plugins.themes.immersion.adminMenu">Administration</message>
+	<message key="plugins.themes.immersion.about.journal">À propos du journal journal</message>
```
Comment (Vitaliy-1) [inline locale/fr_FR/locale.xml]: Thanks. Can you review this one? It should be just  `À propos du journal`, right?

Comment (bastoune) [reply locale/fr_FR/locale.xml]: Hello, review done !

**Comment by Vitaliy-1** [comment]
Thanks!

**Comment by asmecher** [comment]
@Vitaliy-1, don't forget to convert the locale files to the new `.po` format -- you can use `lib/pkp/tools/xmlToPo.php` in your OJS/OMP checkout to do that!

**Comment by asmecher** [comment]
@bastoune, thanks for contributing -- we've moved to a new format/toolset for locale files; see https://pkp.sfu.ca/2020/01/22/introducing-weblate-a-new-path-for-ojs-omp-translations/ for details.

**Pull request merged**